src/styles/catalog.css

```diff
@@ -0,0 +1,12 @@
+body {
+  margin: 0;
+}
+
+.content {
```

Comment:
Consider using BEM naming convention for classes. Rename '.content' to '.catalog' or a similar BEM-style name.